src/Facebook/Facebook.php

```diff
@@ -327,7 +350,7 @@ public function getRedirectLoginHelper()
    */
   public function getJavaScriptHelper()
   {
-    return new FacebookJavaScriptHelper($this->app);
+    return new FacebookJavaScriptHelper($this->app, $this->getOAuth2Client());
```

Comment:
Sounds good! Maybe once we get more eyeballs on the code we'll get some nicer dependency handling going forward. One step at a time! :) I'll refactor and push in a bit.